plugin/src/main/kotlin/com/nishtahir/CargoExtension.kt

```diff
@@ -35,6 +35,7 @@ open class CargoExtension {
     var module: String? = null
     var libname: String? = null
     var targets: List<String>? = null
+    var prebuiltToolchains: Boolean? = null
```

Comment:
So, I know you didn't write this initially, but I think we want to be able to configure this via local.properties as well. You can take a look at the other uses of localProperties in this file for reference. If it's specified in local.properties, I think that should take precedence.

Reply:
Implementing that logic directly in here is a bit awkward - the getter would need to effectively ignore the set value if overridden by localProperties.  Or I could have a separate overriddenPrebuiltToolchains, but that seems easy to overlook when I can't make prebuiltToolchains's getter private in Kotlin without making the setter private too.

RustAndroidPlugin::configurePlugin - the only user of prebuitlToolchains - already looks at localProperties too though, so I can implement local.properties override logic in there without things looking too awkward?

Reply:
> RustAndroidPlugin::configurePlugin - the only user of prebuitlToolchains - already looks at localProperties too though, so I can implement local.properties override logic in there without things looking too awkward?

SGTM, I didn't mean to imply that it needed to be done in the place where I commented, it was just a convenient place to mention it.